src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldMedia.class.php

```diff
@@ -73,12 +74,29 @@ protected function getHtmlRenderedHtml($bReadOnly)
 
             $iWidth = 0;
             $iHeight = 0;
-            if (isset($oImage->aData) && isset($oImage->aData['height'])) {
-                $iHeight = $oImage->aData['height'];
-            }
-            if (isset($oImage->aData) && isset($oImage->aData['width'])) {
-                $iWidth = $oImage->aData['width'];
+            if (isset($oImage->aData)) {
```

Comment:
isset could/should be more specific: ie is_array here? and array_key_exists for the next checks

Reply:
I've only taken the previous code. But I can improve it as suggested.

Reply:
done